docs/modules/ROOT/pages/object-storage/create.adoc

```diff
@@ -29,6 +29,28 @@ spec:
 <4> Secret where the connection details are provisioned. This secret shouldn't exist at this time.
 +
 IMPORTANT: Object Storage Bucket names must be unique across providers. Make sure to choose a name with prefixes and other guards, to prevent any issue during the creation of the bucket. No error message is returned if the bucket cannot be created.
+== Create a Bucket in another cloud provider
+
+By default the bucket is created on the same cloud provider where the cluster is running on.
+For instance, if you are on an Exoscale cluster, to create an ObjectBucket on cloudscale.ch add the following YAML snippet to the `spec` of an ObjectBucket:
+
+[source,yaml]
+----
+spec:
+  compositionRef:
+    name: cloudscale.objectbuckets.appcat.vshn.io <1>
```

Comment:
Where can I find the list of possible / available composition names?

Reply:
Added a new page compositions with the cloud provider references or better compositions. Also linked the page here for better user experience.